File: projectEuler_net/problem_1/problem_1.java
public class problem_1 {
  public static void main(String args[]){
    int result = 0;
    //loop through the numbers
    for(int x = 1; x < 1000; x++){
      //check if number(x) is divisable by 3 or 5
      boolean divisableBy3 = (x % 3 == 0);
      boolean divisableBy5 = (x % 5 == 0);
      if(divisableBy3 || divisableBy5){
        //yes?
          // add x to result
            result = result + x;
        }
        else{
        //no?
          // x + " is not divisable by 3 or 5. "
          //skip to next x
        }
    }

    System.out.println("Result: \n" + result);
  }
}

File: projectEuler_net/problem_1/problem_1.c
#include <stdio.h>
int main(){
    int result = 0;
    int count = 0;
    int max = 1000;
    int divisableBy3;
    int divisableBy5;

    for(count = 1; count < max; count++){
      divisableBy3 = count % 3 == 0 ;
      divisableBy5 = count % 5 == 0 ;
      if(divisableBy3 || divisableBy5){
        result = result + count;
      }
    }
    printf("Result: %d\n", result);
  return 0;
}
